Pull request to correct a mistake in the intro to R on variables.

CristianRiccio opened the pull request:
Hi Mark, 
I am not very comfortable yet with Github but I am trying to ask you to correct a mistake on the naming of variables in the introduction to R. Variables can contain '.' or '_'.

markdunning committed:
version of slides more-compatible with being shown from the markdown script

markdunning committed:
New R notebooks to replace the Day 1 and 2 slides

markdunning committed:
add convenience scripts to generate data, images and zip file

markdunning committed:
Update solutions for Day 1

markdunning committed:
more solution updates

markdunning committed:
small update I missed to exercise 7

markdunning committed:
New solutions zip file

example of adding proportions to count data

markdunning committed:
separate example of cbind and rbind into different chunks

markdunning committed:
Minor grammar change

markdunning committed:
call a script to create the patients data

remove some arguments previously used for slide layout

markdunning committed:
Add some blank code chunks for solutions

Use script to create patients data

markdunning committed:
add new script to create patients data

markdunning committed:
Add files from Jan 2017 run of course using R notebooks

markdunning committed:
Day 1 re-compiled to have contents in HTML notebook

markdunning committed:
Re-compile Day 2

markdunning committed:
Revert exercise 6 to linear regression example

markdunning committed:
re-compile exercise 6

cambioinfo committed:
Merge pull request #9 from cambiotraining/cruk

R notebook presentations

markdunning committed:
- add a missing sentance in the instructions for exercise 5a

markdunning committed:
Correct typos and issues from Feb  2017 course

markdunning commented:
Hi Christian,

So sorry but I'm away at the moment and am not able to look into it. Was it just the problem with the variable names?

CristianRiccio commented:
Hi Mark,
That's the only problem I found so far.

markdunning committed:
correct the path to the images files

markdunning committed:
Add css file to zip (required to preview the notebook)

markdunning committed:
try and fix image files inside course zip

markdunning committed:
Update the link to the spreadsheet lesson from Data Carpentry

markdunning committed:
Add missing images for stats exercise

markdunning committed:
Add Hint about using paste in the stats exercise

Add explanation on how genes were selected to create the heatmap (and pick new set of genes).

markdunning committed:
Re-compile some sections with a larger patient cohort dataset

markdunning committed:
Re-write exercise 3 solution

markdunning committed:
re-generate zip file

markdunning committed:
Hint about producing a data frame in reverse order

markdunning committed:
Fix typo

markdunning committed:
Delete todo

markdunning committed:
add example of pos and adj

markdunning committed:
Merge branch 'master' of https://github.com/cambiotraining/r-intro

markdunning committed:
correct small typo in one code chunk definition

markdunning committed:
add extra hint about changing the names under each box

markdunning committed:
Add note about different solution to exercise 5 where a single vector of colours and plotting characters is used rather than points

markdunning committed:
small change to where axis labels are printed in axis example

markdunning committed:
Change colour for the wind speed histogram to avoid possible clash with red normal curve

markdunning committed:
Move matrices examples to the end of section

Remove the variables used to create the data frame. Hopefully encourage people to subset using the actual columns from the data frame

markdunning committed:
Remove example of subsetting the matrix e (which no longer exists)

markdunning committed:
correct a bad variable name

markdunning committed:
- add a couple of extra explanation sentances

- do a match by selecting the symbol column by name rather than index

- replace t variable name

- remove sample names from heatmap

ruidlpm committed:
clarity of use parameter in cor function

In the cor function, instead of using the abbreviation use="c", it is clearer to write use="complete.obs".

tavareshugo committed:
Merge pull request #23 from ruidlpm/patch-1

clarity of use parameter in cor function
